tokio/src/sync/mpsc/unbounded.rs

```diff
@@ -162,6 +162,10 @@ impl<T> UnboundedSender<T> {
     }
 
     /// Attempts to send a message on this `UnboundedSender` without blocking.
+    ///
+    /// If the receive half of the channel is closed, either due to [`close`](UnboundedReceiver::close)
+    /// being called or the [`UnboundedReceiver`](UnboundedReceiver) handle dropping, the function returns
+    /// an error. The error includes the value passed to `send`.
```

Comment:
```suggestion
    /// If the receive half of the channel is closed, either due to [`close`](UnboundedReceiver::close)
    /// being called or the [`UnboundedReceiver`](UnboundedReceiver) handle having been dropped,
    /// the function returns an error. The error includes the value passed to `send`.
```
The phrase "or the `UnboundedReceiver` handle dropping" is a bit awkward.

Reply:
@Darksonn Honestly, I copied that part from `Sender::send()` doc, perhaps it should be changed there too...

Reply:
If you want, you can update it both places in this PR.

Reply:
Ok, it's done :)